app/components/agent/Jobs/index.js

```diff
@@ -47,7 +47,7 @@ class AgentJobs extends React.PureComponent<Props, State> {
     loading: false
   };
 
-  _agentRefreshTimeout: number;
+  _agentRefreshTimeout: TimeoutID;
```

Comment:
Ooh! Is this new? I like it. Giving names to previously opaque concepts!!!!! Awesome!

Reply:
Yeah, it had a cry at me because we were using the wrong return type for timeouts. Much nicer!